concourse/Dockerfile

```diff
@@ -0,0 +1,15 @@
+ARG CONCOURSE_VERSION=5.8.0
+
+FROM alpine:latest as downloader
```

Comment:
what's the benefit of doing it like this vs using one single `FROM`?

Reply:
by doing it like this you dispose the container that is used for the downloading and you only install what you actually need in the final container

Reply:
I see, interesting!